sound/soc/sof/intel/hda.h

```diff
@@ -526,6 +527,27 @@ int hda_ipc_pcm_params(struct snd_sof_dev *sdev,
 		       struct snd_pcm_substream *substream,
 		       const struct sof_ipc_pcm_params_reply *reply);
 
+/*
+ * Probe Compress Operations.
+ */
+unsigned int hda_compr_stream_tag(struct snd_sof_dev *sdev,
+		struct snd_compr_stream *cstream);
+int hda_probe_compr_open(struct snd_sof_dev *sdev,
+		struct snd_compr_stream *cstream,
+		struct snd_soc_dai *dai);
+int hda_probe_compr_free(struct snd_sof_dev *sdev,
+		struct snd_compr_stream *cstream,
+		struct snd_soc_dai *dai);
+int hda_probe_compr_set_params(struct snd_sof_dev *sdev,
+		struct snd_compr_stream *cstream,
+		struct snd_compr_params *params, struct snd_soc_dai *dai);
+int hda_probe_compr_trigger(struct snd_sof_dev *sdev,
+		struct snd_compr_stream *cstream, int cmd,
+		struct snd_soc_dai *dai);
+int hda_probe_compr_pointer(struct snd_sof_dev *sdev,
+		struct snd_compr_stream *cstream,
+		struct snd_compr_tstamp *tstamp, struct snd_soc_dai *dai);
```

Comment:
same here, kernel styles please (checkpatch.pl --strict --codespell)

Reply:
Fixed in v6.